### ISSUE TITLE
Delete Cookie Savestate

### ISSUE THREAD

**Issue by edutac** [rstaib/jquery-steps#137]
Hi, congratulations for your great work ! Just a question about the savestate. How can I remove the cookie which stores the last active step once I click on the Finish button ?
Thanks for your answer !

**Comment by EnigmaSolved** [comment]
You probably just need to use the appropriate delete method in [js-cookie](https://github.com/js-cookie/js-cookie), and call that in your `onFinished` function call (https://github.com/rstaib/jquery-steps/wiki/Settings#events).  If you weren't already aware, the js-cookie plugin is required in order for `saveState` to work (https://github.com/rstaib/jquery-steps/wiki/Requirements).

**Comment by icemanblas** [comment]
Hi, first of all, congratilations for the plugin, it's great.
Since I have the same problem, can someone post the solution, or provide a little bit more details on this issue.
Thanks in advance.

**Comment by edutac** [comment]
Hi, you can follow the advice from EnigmaSolved (thanks !) but if you want to manually delete the cookie, you have to find his name. In my case the cookie file was jQu3ry_5teps_St%40te_steps-uid-0 
To delete in php : setcookie('jQu3ry_5teps_St%40te_steps-uid-0', NULL, -1);

**Comment by icemanblas** [comment]
Yeah, but my idea was to do it dinamically, not by providing a string with the cookie name... Generally, the solution works well.
Thanks

**Comment by edutac** [comment]
Sorry see the advice of EnigmaSolved, I can't help you more :-(
